src/lib/util.js

```diff
@@ -1026,14 +1026,31 @@ function getInterfaceTreeInfo(iface, interfaces) {
  *   if the requested schema does not exist.
  */
 function getSchemaValidationFunction(schemaName) {
+    // Helper function that selects the right schema file from the given
```

Comment:
you may question-mark as much as you want, it actually does improve readability :)